events/rateLimit.js

```diff
@@ -0,0 +1,12 @@
+const { Event } = require('klasa');
+
+const HEADER = `\x1b[31m[RATELIMIT]\x1b[0m`;
+
+module.exports = class extends Event {
+
+	run({ timeout, limit, method, path, route }) {
+		this.client.emit('verbose',
+			`${HEADER} \nTimeout: ${timeout}ms\nLimit: ${limit} requests\nMethod: ${method.toUpperCase()}\nPath ${path}\nRoute: ${route}`);
```

Comment:
1. Can me shortened using an array (I'm pretty sure they'll be joined by newline).
1. Path and route are identical, use one of them.